docs/zhCn/md/step-forward-docs/advanced-configuration.md

```diff
@@ -34,3 +34,12 @@ cql:
     statement-start: <% # 默认为 @
     statement-end: %> # 默认为 null
 ```
+
+## 使用自定义的 session存活有效期/健康检测间隔(单位毫秒)
+```yml
+cql:
+  parser:
+    # 建议少于nebula配置文件中失效时间
+    session-life-length: 18000000
+    check-fixed-rate: 3600000

```

Comment:
This scope is more suitable for xml-related configuration. 
Is it more suitable to use `nebula.ngbatis.session-life-length`? 
The interface does not need additional parameters.
It can be obtained through NebulaPoolConfig.

Reply:
The field extension of com.vesoft.nebula.client.graphic.NebulaPoolConfig is not very convenient

Reply:
Do I need to add a new configuration class

Reply:
Sorry, I made a mistake.
Could you consider creating a session scope in NebulaJdbcProperties?

Reply:
Or use nebula-java SessionPool directly ?

Reply:
> Sorry, I made a mistake. Could you consider creating a session scope in NebulaJdbcProperties?

The ngbatis field is added in NebulaJdbcProperties to customize the ngbatis parameter.

Reply:
> Or use nebula-java SessionPool directly ?

This is also a feasible solution, but com.vesoft.nebula.client.graphic.SessionPool isSessionError may need to be adjusted?
`private boolean isSessionError(ResultSet resultSet) {
        return resultSet != null
                && (resultSet.getErrorCode() == ErrorCode.E_SESSION_INVALID.getValue()
                || resultSet.getErrorCode() == ErrorCode.E_SESSION_NOT_FOUND.getValue()
                || resultSet.getErrorCode() == ErrorCode.E_SESSION_TIMEOUT.getValue());
}`

When resultSet is null, the session needs to be released so that the new session can be obtained after the nebula service is restarted.

Reply:
> Or use nebula-java SessionPool directly ?

use nebula-java SessionPool with switch config